### PR TITLE
Change the print helm chart resources YAML command name keadm generate to keadm manifest.

### PR DESCRIPTION
**Pull request by luomengY**
… to keadm manifest.

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
image
**Which issue(s) this PR fixes**:

Fixes A part of #5317

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by wbc6080 — COMMENTED** [review]
Seems this PR and 5463, 5461 are aimed to solve #5317 together, so you'd better change the "Which issue(s) this PR fixes:" content in PR description. Otherwise, any one of these PRs after being merged will cause the issue to be closed . Maybe you can use "A part of #5317"

**Comment by luomengY** [comment]
> Seems this PR and 5463, 5461 are aimed to solve #5317 together, so you'd better change the "Which issue(s) this PR fixes:" content in PR description. Otherwise, any one of these PRs after being merged will cause the issue to be closed . Maybe you can use "A part of #5317"

ok

File: keadm/cmd/keadm/app/cmd/cloud/manifest.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func NewManifestGenerate() *cobra.Command {
 	flagVals := make(map[string]types.FlagData)
 
 	var generateCmd = &cobra.Command{
-		Use:     "generate",
+		Use:     "manifest",
 		Short:   "Checks and generate the manifests.",
 		Long:    cloudManifestGenerateLongDescription,
```
Comment (WillardHu) [inline keadm/cmd/keadm/app/cmd/cloud/manifest.go:69]: Modify and clean up long messages

**Review by WillardHu — CHANGES_REQUESTED** [review]

File: keadm/cmd/keadm/app/cmd/cloud/manifest.go (hunk 1 above)
Comment (luomengY) [reply keadm/cmd/keadm/app/cmd/cloud/manifest.go]: I have Modified it.

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**